Assets/Scripts/Portal.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class Portal : MonoBehaviour
+{
+	// Start is called before the first frame update
+	public GameObject portalOpposite;
+	Vector3 distanceToExit;
+	void Start()
+	{
+		distanceToExit = portalOpposite.transform.position - transform.position;
+	}
+
+	void OnTriggerEnter(Collider c)
+	{
+		Debug.Log(c.gameObject.tag);
+		Debug.Log("distance: " + distanceToExit);
+		if (c.gameObject.tag == "Player")
+		{
+			c.GetComponent<CharacterController>().enabled = false;
+			c.transform.position = c.transform.position + distanceToExit * 1.05f;
```

Comment:
is 1.05f so that we're slightly in front of the portal

Reply:
yes